fuzzers/qemu_systemmode/src/fuzzer_classic.rs

```diff
@@ -5,7 +5,7 @@ use std::{env, path::PathBuf, process};
 
 use libafl::{
     corpus::{Corpus, InMemoryCorpus, OnDiskCorpus},
-    events::{launcher::Launcher, EventConfig},
+    events::{launcher::Launcher, EventConfig, CTRL_C_EXIT},
```

Comment:
Is this change the failing qemu thingy?

Reply:
Yes.

Reply:
Then we should revert this file to main

Reply:
can you do that easily?

Reply:
git checkout fuzzer_classic.rs -- origin/main

Reply:
I'd do it the other way around (not sure if both work? :D )

so
`git checkout main -- ./fuzzers/qemu_systemmode/src/fuzzer_classic.rs`

Reply:
Done